snuba/query/conditions.py

```diff
@@ -81,6 +74,21 @@ def __set_condition_pattern(
     )
 
 
+set_condition_pattern = {
+    op: __set_condition_pattern(AnyExpression(), op) for op in FUNCTION_TO_OPERATOR
+}
+
+
+def __is_set_condition(exp: Expression, operator: str) -> bool:
+    if is_binary_condition(exp, operator):
+        if operator in set_condition_pattern:
+            if set_condition_pattern[operator].match(exp) is not None:
+                # TODO(evanh) Not sure how to test this using the current matcher syntax
+                return all(isinstance(c, Literal) for c in exp.parameters[1].parameters)
```

Comment:
Doesn't mypy complain here? It seems it should not know that exp is a FunctionCall and thus have a `parameters` attribute.